tests/acceptance/pageObjects/FilesPageElement/filesRow.js

```diff
@@ -3,60 +3,6 @@ const filesList = client.page.FilesPageElement.filesList()
 
 module.exports = {
   commands: {
-    /**
```

Comment:
we are getting to the point that the fileRow does not do anything much by itself, need to think if we want to refactor that and get rid of that PageObject (not in this PR)